plugins/chat/assets/javascripts/discourse/initializers/chat-decorators.js

```diff
@@ -1,4 +1,4 @@
-import { decorateGithubOneboxBody } from "discourse/initializers/onebox-decorators";
+import { decorateGithubOneboxBody } from "discourse/instance-initializers/onebox-decorators";
```

Comment:
This assumes:

1. This is the canonical source for the `chat` plugin (i.e. it's not a vendored copy from somewhere else)
2. The initializer modules aren't meant to be public APIs and this is the only place that does it

Otherwise we'll have to make a no-op initializer in `discourse/initializers/onebox-decorators` to re-export the function to preserve compatibility

Reply:
I've checked all the theme/plugin codebases we're aware of, and I don't see any occurrences of the string 'onebox-decorators', so I think we're good to proceed here 👍